packages/widget/src/controllers/wallet-manager/manager.ts

```diff
@@ -7,23 +7,66 @@ import Onboard from '@web3-onboard/core';
 import injectedModule from '@web3-onboard/injected-wallets';
 import walletConnectModule from '@web3-onboard/walletconnect';
 import type { ReactiveController, ReactiveElement } from 'lit';
+import type { WalletInit, AppMetadata } from '@web3-onboard/common';
 
 import type { WalletConnectOptions } from '@web3-onboard/walletconnect/dist/types';
-import type { AppMetadata } from '@web3-onboard/common';
 import { utils } from 'ethers';
-import { WalletUpdateEvent, walletContext } from '../../context';
+import { configContext, WalletUpdateEvent, walletContext } from '../../context';
 
 export class WalletController implements ReactiveController {
   host: ReactiveElement;
 
   walletContext: ContextConsumer<typeof walletContext, ReactiveElement>;
+  _configContext: ContextConsumer<typeof configContext, ReactiveElement>;
+
+  /**
+   * Provides list of wallets specified by 3rd party
+   * along with default injected connector
+   * @param {{ dappUrl?: string }} options
+   * @returns {WalletInit[]}
+   */
+  getWallets(options?: {
+    walletConnectOptions?: WalletConnectOptions;
+    appMetaData?: AppMetadata;
```

Comment:
instead of subscribing to config context, you can just add `walletSelectorWalletConfigurations` here